pcsx2/CDVD/CDVDcommon.h

```diff
@@ -65,6 +94,12 @@ typedef struct _cdvdTN
 #define CDVD_TYPE_DETCT 0x01 // Detecting
 #define CDVD_TYPE_NODISC 0x00 // No Disc
 
+// SUBQ CONTROL:
+#define AUDIO_PREEMPHASIS(control) ((control & (4 << 1)))
```

Comment:
If these are intended to be used as true/false helpers. I would rather them be named stuff like CDVD_IS_DIGITAL_COPY, CDVD_IS_DATA, CDVD_HAS_QUADRAPHONIC_AUDIO, or whatever makes sense but with that intent.

It makes the usage of those a little more clear and it also doesn't populate any files that include this file (this is a common file) with a bunch of macros without a namespace.